lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -569,6 +569,7 @@
     <datatype extension="ttl" type="galaxy.datatypes.triples:Turtle" display_in_upload="true"/>
     <datatype extension="rdf" type="galaxy.datatypes.triples:Rdf" display_in_upload="true"/>
     <datatype extension="jsonld" type="galaxy.datatypes.triples:Jsonld" display_in_upload="true"/>
+    <datatype extension="sbol" type="galaxy.datatypes.triples:Sbol" display_in_upload="true"/>
```

Comment:
Can you add a description? Three is a description attribute.

Reply:
Sure, done in 3e6899163ccccc7a656877fcdc04060f44fba799